eng/scripts/Sdk-Generate.ps1

```diff
@@ -0,0 +1,173 @@
+param(
+$GenerateInput, 
+$GenerateOutput,
+$RepoRoot = "${PSScriptRoot}/../.."
+)
+
+$input = Get-Content $GenerateInput | ConvertFrom-Json
+$inputFiles = $input.changedFiles;
+$inputFiles += $input.relatedReadmeMdFiles;
+$inputFiles = $inputFiles | select -Unique
+Write-Output "List Of changed swagger files and related readmes" $inputFiles "`n"
+
+$autorestFilesPath = Get-ChildItem -Path "$RepoRoot/sdk"  -Filter autorest.md -Recurse | Resolve-Path -Relative
+
+Write-Host "Updating autorest.md files for all the changed swaggers."
+$sdkPaths = @()
+$headSha = $input.headSha
+$repoHttpsUrl = $input.repoHttpsUrl
+foreach ($inputFile in $inputFiles)
+{
+  foreach ($path in $autorestFilesPath)
+  {
+    $fileContent = Get-Content $path
+    $isUpdatedLines = $false
+    $updatedLines = foreach($line in $fileContent)
+    {
+      if($line -match "[\/][0-9a-f]{4,40}[\/]$inputFile")
+      {
+        # replacing sha
+        $line = $line -replace "[\/][0-9a-f]{4,40}[\/]$inputFile", "/$headSha/$inputFile"
+        # replacing repo path
+        $line -replace "https:\/\/[^`"]+?$headSha", "$repoHttpsUrl/blob/$headSha"
+
+        $isUpdatedLines = $true
+        $sdkPaths += (get-item $path).Directory.Parent.FullName | Resolve-Path -Relative
+      }
+      else
+      {
+        $line
+      }
+    }
+    if($isUpdatedLines)
+    {
+      $updatedLines | Out-File -FilePath $path
+    }
+  }
+}
+
+Write-Host "Updated autorest.md files for all the changed swaggers. `n"
+
+function Test-PreviousScript() 
+{
+  $result = $null
+  if($LASTEXITCODE -eq 0)
+  {
+    $result = "succeeded"
+  }
+  else 
+  {
+    $result = "failed"
+  }
+  Write-Output $result
+}
+
+$sdkPaths = $sdkPaths | select -Unique
+$packages = @()
+$artifactsPath = "$RepoRoot/artifacts/packages"
+if(!(Test-Path $artifactsPath))
+{
+  New-Item -ItemType Directory -Force -Path $artifactsPath
```

Comment:
Why do you need to create this? Running the pack commands should end up creating it.